api.planx.uk/modules/webhooks/service/sendNotification/index.ts

```diff
@@ -36,7 +36,11 @@ export const sendSlackNotification = async (
   const pilotServices = [
     "uniform",
     "happ",
+    "hapr",
+    "hret",
     "ldc",
+    "minor",
+    "major",
```

Comment:
again, I don't _think_ minors & majors are actually live in pilot, but just in case one is sent to BOPS we'll now properly flag it with the :large_orange_diamond: